airflow/providers/amazon/aws/transfers/dynamodb_to_s3.py

```diff
@@ -84,15 +84,20 @@ class DynamoDBToS3Operator(AwsToAwsBaseOperator):
 
     :param dynamodb_table_name: Dynamodb table to replicate data from
     :param s3_bucket_name: S3 bucket to replicate data to
+    :param s3_bucket_owner: The ID of the Amazon Web Services account that owns the bucket the export will be stored in.
```

Comment:
I think we can remove this one. This one can be provider as part of `export_table_to_point_in_time_kwargs`

Reply:
sounds good! made the change